src/spatialdata_plot/pl/basic.py

```diff
@@ -398,52 +660,55 @@ def render_images(
+        palette: list[str] | str | None = None,
         cmap: Colormap | str | None = None,
-        norm: None | Normalize = None,
-        na_color: str | tuple[float, ...] | None = (0.0, 0.0, 0.0, 0.0),
-        outline_alpha: float = 1.0,
-        fill_alpha: float = 0.3,
-        scale: str | list[str] | None = None,
+        norm: Normalize | None = None,
+        na_color: ColorLike | None = (0.0, 0.0, 0.0, 0.0),
+        outline_alpha: float | int = 1.0,
+        fill_alpha: float | int = 0.3,
+        scale: list[str] | str | None = None,
         **kwargs: Any,
     ) -> sd.SpatialData:
         """
         Render labels elements in SpatialData.
 
         Parameters
         ----------
-        elements
-            The name of the labels element(s) to render. If `None`, all
-            labels elements in the `SpatialData` object will be used.
-        color
+        elements : list[str] | str | None, optional
+            The name(s) of the label element(s) to render. If `None`, all label
+            elements in the `SpatialData` object will be used.
+        color : str | None, optional
             Key for annotations in :attr:`anndata.AnnData.obs` or variables/genes.
-        groups
-            For discrete annotation in ``color``, select which values
-            to plot (other values are set to NAs).
-        contour_px
+        groups : list[str] | str | None, optional
+            When using `color` and the key represents discrete labels, `groups`
+            can be used to show only a subset of them. Other values are set to NA.
+        contour_px : int, default 3
             Draw contour of specified width for each segment. If `None`, fills
             entire segment, see :func:`skimage.morphology.erosion`.
-        outline
+        outline : bool, default False
             Whether to plot boundaries around segmentation masks.
-        layer
+        layer : str | None, optional
             Key in :attr:`anndata.AnnData.layers` or `None` for :attr:`anndata.AnnData.X`.
-        palette
-            Palette for discrete annotations, see :class:`matplotlib.colors.Colormap`.
-        cmap
+        palette : list[str] | str | None, optional
+            Palette for discrete annotations. List of valid color names that should be
+            used for the categories. Must match the number of groups.
+        cmap : Colormap | str | None, optional
             Colormap for continuous annotations, see :class:`matplotlib.colors.Colormap`.
-        norm
+        norm : Normalize | None, optional
             Colormap normalization for continuous annotations, see :class:`matplotlib.colors.Normalize`.
-        na_color
+        na_color : ColorLike | None, optional
             Color to be used for NAs values, if present.
-        alpha
+        outline_alpha : float | int, default 1.0
+            Alpha value for the labels.
+        fill_alpha : float | int, default 0.3
             Alpha value for the labels.
```

Comment:
maybe add outline or fill respectively so that we don't have the same description twice?